lib/kino/proxy.ex

```diff
@@ -29,12 +29,10 @@ defmodule Kino.Proxy do
   Using the proxy feature, we can use Livebook apps to build APIs.
   For example, we could provide a data export endpoint:
 
-      data = <<...>>
-      token = "auth-token"
 
       Kino.Proxy.listen(fn
         %{path_info: ["export", "data"]} = conn ->
-          ["Bearer " <> ^token] = Plug.Conn.get_req_header(conn, "authorization")
+          data = "some data"
```

Comment:
In this case, using bearer tokens may be the best example, since tokens are most common for APIs (and Kino.Proxy would most likely be used for APIs?)

Reply:
Indeed.

I'll change from HTTP basic auth to HTTP Bearer.

Reply:
I thought about leaving this example as simple as possible, with no auth. And the auth would be below, inside the admonition block.